plugins/filebrowser/plugin.js

```diff
@@ -604,21 +604,20 @@
  *
  * Available values:
  *
- *	* `'xhr'` - XMLHttpRequest is used to upload file. Using this option allows to set up with Additional XHR headers with
+ *	* `'xhr'` - (default) XMLHttpRequest is used to upload file. Using this option allows to set up with additional XHR headers with
  * {@link CKEDITOR.config#fileTools_requestHeaders} option.
- *	* `'form'` - (default) File is uploaded by submitting a traditional `<form>` element.
+ *	* `'form'` - File is uploaded by submitting a traditional `<form>` element.
  *	* `null` - The default method is used.
```

Comment:
Please add a remark that until version 4.9.0 `form` option was the only option (thus it was) used by CKEditor.